src/app/cases/viewer/case-viewer.component.html

```diff
@@ -38,10 +38,10 @@ <h3 class="heading-medium error-summary-heading" id="edit-case-event_error-summa
                   <th>
                     <div class="case-viewer-label">{{field.label}}</div>
                   </th>
-                  <td><ccd-field-read [caseField]="field"></ccd-field-read></td>
+                  <td><ccd-field-read [caseField]="field" [caseViewContext]="buildContextMap()"></ccd-field-read></td>
```

Comment:
Those are not the only places where `ccd-field-read` is used. Because `caseViewContext` appears to be a mandatory input, it should be added to every place where `ccd-field-read` is used (read only in wizard, check your answers, case list, search results...).
Incidentally, this requires modifying a few areas that are not directly related to the current change.

Reply:
Do you mean mandatory as in being setup inside FieldReadComponent? If so I think it will not be used as it is not defined for those other places (read only in wizard, check your answers, case list, search results...). Therefore not sure this would require changes in other areas as I felt this is like optional and can be skipped meaning it's not being used.